libs/partners/milvus/tests/integration_tests/vectorstores/test_milvus.py

```diff
@@ -303,6 +304,32 @@ def test_milvus_enable_dynamic_field_with_partition_key() -> None:
     }
 
 
+def test_milvus_sparse_embeddings() -> None:
+    texts = [
+        "In 'The Clockwork Kingdom' by Augusta Wynter, a brilliant inventor discovers "
+        "a hidden world of clockwork machines and ancient magic, where a rebellion is "
+        "brewing against the tyrannical ruler of the land.",
+        "In 'The Phantom Pilgrim' by Rowan Welles, a charismatic smuggler is hired by "
+        "a mysterious organization to transport a valuable artifact across a war-torn "
+        "continent, but soon finds themselves pursued by assassins and rival factions.",
+        "In 'The Dreamwalker's Journey' by Lyra Snow, a young dreamwalker discovers "
+        "she has the ability to enter people's dreams, but soon finds herself trapped "
+        "in a surreal world of nightmares and illusions, where the boundaries between "
+        "reality and fantasy blur.",
+    ]
+    sparse_embedding_func = BM25SparseEmbedding(corpus=texts)
+    docsearch = Milvus.from_texts(
+        embedding=FakeEmbeddings(),
+        texts=texts,
+        sparse_embedding_function=sparse_embedding_func,
+        connection_args={"uri": "./milvus_demo.db"},
+        drop_old=True,
+    )
+
+    output = docsearch.similarity_search("Pilgrim", k=1)
+    assert "Pilgrim" in output[0].page_content
+
+
 # if __name__ == "__main__":
```

Comment:
could you please add "#     test_milvus_sparse_embeddings()" to main(), although it is a comment. Just for completeness and suggestion

Reply:
Done